cloudformation/deploy/template.yaml

```diff
@@ -707,7 +707,7 @@ Resources:
             - Name: SUPPORT_CHECK_EMAIL_FRAUD
               Value: 1
             - Name: SUPPORT_MULTI_CHANNEL
-              Value: 0
+              Value: 1
```

Comment:
Are we confident that this will roll out to all environments before we cut over to the new frontend?

Reply:
Thanks @ethanmills - I'll double-check with @monhaque (when they have returned from leave) and make any necessary changes.